demos/ag-alert-backend-lastbyte-resp/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "itemDisplayName": "Create an alert using Backend Last Byte Response metric for your Application Gateway.",

```

Comment:
see pipeline result for changes that need to be made here

Reply:
done